react-stopwatch/my-app/src/Stopwatch.tsx

```diff
@@ -0,0 +1,62 @@
+import { FaPause, FaPlay } from 'react-icons/fa';
+import './Stopwatch.css';
+import { useState, useEffect } from 'react';
+
+export function Stopwatch() {
+  const [elapsedSeconds, setElapsedSeconds] = useState(0);
+  const [isRunning, setIsRunning] = useState(false);
+  const [intervalId, setIntervalId] = useState<NodeJS.Timeout | null>(null);
+
+  useEffect(() => {
```

Comment:
We haven't learned `useEffect` yet, so this can be solved without it. However, your solution works! And you got to learn something we haven't studied yet. :-)

We will get into more detail on useEffect in a couple of weeks.